crd/nodenetworkconfig/client.go

```diff
@@ -26,6 +28,12 @@ func init() {
 	_ = v1alpha.AddToScheme(Scheme)
 }
 
+type OwnerExistsError struct{}
```

Comment:
can we flip this around to ErrOwnerExists? seems more canonical (https://go.dev/src/os/error.go) even though k/k likes to do "ThingError"

Reply:
_if_ we still need it instead of using ctrlutil's similar defined error

Reply:
removed, caller can rely on ctrlutil's error with `errors.Is/As` since we just wrap it